pytest_splunk_addon/standard_lib/addon_parser/eventtype_parser.py

```diff
@@ -13,11 +13,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-# -*- coding: utf-8 -*-
 """
 Provides eventtypes.conf parsing mechanism
 """
+from typing import Dict
```

Comment:
My preference would be 
`from typing import Dict,Generator,Optional`

Reply:
I will onboard `isort` soon, so we don't need to have preferences regarding the style.